composer.json

```diff
@@ -40,6 +40,7 @@
     },
     "require-dev": {
         "ext-eio": "^2",
+        "ext-posix": "*",
```

Comment:
We shouldn't require `ext-posix` here, because we also support Windows.

Reply:
Should I remove the fifo files from tests then?

Reply:
There's no reason to remove tests, we just need to skip them if `ext-posix` isn't available.